integration-tests/src/test/scala/org/constellation/schema/ClusterNode.scala

```diff
@@ -0,0 +1,11 @@
+package org.constellation.schema
+
+import io.circe.generic.semiauto.{deriveDecoder, deriveEncoder}
+import io.circe.{Decoder, Encoder}
+
+case class ClusterNode(alias: String, id: Id, ip: HostPort, status: NodeState, reputation: Long)
```

Comment:
@marcinwadon @kpudlik This case class is copied from the node, because I couldn't find it in schema. Should I move it from node to schema and remove this copy?

Reply:
Can't you take it from the node directly anyway?

Reply:
I don't have a dependency on the node.